Analytics/Classes/SEGAnalytics.m

```diff
@@ -169,12 +169,8 @@ - (void)_applicationWillEnterForeground
     if (!self.configuration.trackApplicationLifecycleEvents) {
         return;
     }
-    NSString *currentVersion = [[NSBundle mainBundle] infoDictionary][@"CFBundleShortVersionString"];
-    NSString *currentBuild = [[NSBundle mainBundle] infoDictionary][@"CFBundleVersion"];
     [self track:@"Application Opened" properties:@{
-        @"from_background" : @YES,
-        @"version" : currentVersion ?: @"",
-        @"build" : currentBuild ?: @"",
+        @"from_background" : @YES
```

Comment:
This should still collect the properties outlined in https://segment.com/docs/spec/mobile/#application-opened